src/org/freedesktop/gstreamer/meta/GstVideoTimeCodeMeta.java

```diff
@@ -0,0 +1,55 @@
+package org.freedesktop.gstreamer.meta;
+
+import com.sun.jna.Pointer;
+import org.freedesktop.gstreamer.MiniObject;
+import org.freedesktop.gstreamer.glib.Natives;
+import org.freedesktop.gstreamer.timecode.GstVideoTimeCode;
+import static org.freedesktop.gstreamer.lowlevel.GstMetaApi.GST_META_API;
+import static org.freedesktop.gstreamer.lowlevel.GstMetaApi.GstVideoTimeCodeMetaStruct;
+
+/**
+ * Extra buffer metadata describing the GstVideoTimeCode of the frame.
+ * @author Jokertwo
+ * @see <a href="https://docs.gstreamer.com/documentation/video/gstvideometa.html?gi-language=c#GstVideoTimeCodeMeta">GstVideoTimeCodeMeta</a>
+ */
+public class GstVideoTimeCodeMeta extends MiniObject {
```

Comment:
VideoTimeCodeMeta. Is this really a MiniObject?

Reply:
I do not know, how to resolve if is it MiniObject or not? 
* [Documentation](https://gstreamer.freedesktop.org/documentation/video/gstvideometa.html?gi-language=c#GstVideoTimeCodeMeta)
* [Source code](https://gitlab.freedesktop.org/gstreamer/gst-plugins-base/-/blob/1.10/gst-libs/gst/video/gstvideotimecode.h#L73)